File: Data Structures/Arrays/removeEven.go
package main

import "fmt"


func removeEven(arr[] int) []int{
	var result int

	for _,ele := range arr{
		if ele%2!=0 {
			result = append(result,ele)
		}
	}
	return result
}

func main(){
	arr:=[]int{1,2,3,4,5,6,7,8}
	newArr := removeEven(arr)
	fmt.Println(newArr)

}

File: Data Structures/Arrays/removeEven.java
import java.util.*;

class CheckRemoveEven {

    public static int[] removeEven(int[] arr) {

        List<Integer> result = new ArrayList<>();

        for (int ele : arr) {
            if (ele % 2 != 0) {
                result.add(ele);
            }
        }

        // Convert ArrayList to array
        int[] newArray = new int[result.size()];
        for (int i = 0; i < result.size(); i++) {
            newArray[i] = result.get(i);
        }

        return newArray;

    }
}
